### ISSUE TITLE
Series doesn't have an is_unique attribute

### ISSUE THREAD

**Issue by scls19fr** [pydata/pandas#11946]
Hello,

Series doesn't have is a `is_unique` attribute

Index have a `is_unique` attribute but not (values of a) Series

```python
>>> df.index.is_unique
True

>>> df['Column1'].is_unique
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-417-c450d4c4ceda> in <module>()
----> 1 df['Date'].is_unique

//anaconda/envs/py27/lib/python2.7/site-packages/pandas/core/generic.pyc in __getattr__(self, name)
   2358                 return self[name]
   2359             raise AttributeError("'%s' object has no attribute '%s'" %
-> 2360                                  (type(self).__name__, name))
   2361
   2362     def __setattr__(self, name, value):

AttributeError: 'Series' object has no attribute 'is_unique'
```

I'm doing this

```python
def is_unique(ser):
    return len(np.unique(ser.values)) == len(ser.values)
```

but this kind of attribute could be useful when you want to set a column as index and ensure that there is no duplicate keys.

PS: I know there is

```python
    df.set_index('Column1', verify_integrity=True)
```

**Comment by jreback** [comment]
``.nunique()`` and ``.unique()`` already exist on a ``Series``, so no need to hand-roll.

I suppose ``.is_unique`` could be defined as the below.

want to do a PR?

```
In [9]: s = Series(np.random.randint(0,10,size=1000))

In [10]: s.nunique() == len(s)
Out[10]: False

In [11]: s = Series(np.arange(1000))

In [12]: s.nunique() == len(s)
Out[12]: True
```

**Comment by jreback** [comment]
closed by #11948
